engine/src/main/java/org/terasology/logic/console/commands/CoreCommands.java

```diff
@@ -506,24 +526,91 @@ public String spawnBlock(@Sender EntityRef sender, @CommandParam("blockName") St
         spawnPos.add(offset);
 
         BlockFamily block = blockManager.getBlockFamily(blockName);
-        if (block == null) {
-            return "";
-        }
 
         BlockItemFactory blockItemFactory = new BlockItemFactory(entityManager);
         EntityRef blockItem = blockItemFactory.newInstance(block);
 
         blockItem.send(new DropItemEvent(spawnPos));
         return "Spawned block.";
     }
+    @Command(shortDescription = "Drops many blocks as per value choosen by the player", helpText = "the particular block can be seleted which the player wants to drop"
+                , runOnServer = true, requiredPermission = PermissionManager.CHEAT_PERMISSION)
+    public String bulkDropper(@Sender EntityRef sender, @CommandParam("blockName") String blockName, @CommandParam("value") int value) {
```

Comment:
Maybe a better name: `bulkDrop`

Seems more natural to `bulkDrop dirt 250` than `bulkDropper dirt 250` and it is slightly shorter